README.md

```diff
@@ -10,6 +10,9 @@ Check out the
 [Yarn Workspaces documentation](https://classic.yarnpkg.com/lang/en/docs/workspaces/)
 for more.
 
+### Analytics
+Read more about our analytics practices [here](ANALYTICS.md).
+
```

Comment:
[nit] I would say this is not a monorepo tip or trick, maybe it could be moved above this section with heading level 2